region/norfair/west/Crocomire Speedway.json

```diff
@@ -239,6 +253,63 @@
         "h_canHeatedCrystalFlash"
       ]
     },
+    {
+      "link": [2, 2],
+      "name": "Croc Speedway Speed Block Moondance (Leave with Stored Fall Speed)",
```

Comment:
Could it also be possible to leave with a door lock bypass at node 3? I remember seeing a clip showing you could turn around while falling down short distances to avoid losing the stored speed, though I don't know if it works here

Reply:
I put a fixme up in links.  It looks like you do it by running at full speed and turning around so that you face forward about when you reach the drop off then momentum carries you down the stairs.  I got it once without (probably) losing stored speed, so it felt frame perfect.